src/test/java/com/stripe/model/BalanceTest.java

```diff
@@ -0,0 +1,91 @@
+package com.stripe.model;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.exception.StripeException;
+import com.stripe.model.Balance;
+import com.stripe.model.Money;
+import com.stripe.net.APIResource;
+import com.stripe.net.LiveStripeResponseGetter;
+import com.stripe.net.RequestOptions.RequestOptionsBuilder;
+import com.stripe.net.RequestOptions;
+import org.junit.After;
```

Comment:
Mind just re-organizing these imports so that the `org.junit` stuff lives with the other import of the same kind below?